### PR TITLE
Add order attribute to project index page

### PR DESCRIPTION
**Pull request by pksarguru**
@LoganDSPrice You were using `.eager_load` in the projects controller.  That wasn't able to pick up repositioning of the projects in the refinery admin page.  I changed it to `.order` since it seems that `.eager_load` doesn't affect how the tags are populated since those are through associations.  What do you think?

### CODE REVIEW

File: vendor/extensions/projects/app/controllers/refinery/projects/projects_controller.rb [hunk 1]
```diff
@@ -8,10 +8,11 @@ class ProjectsController < ::ApplicationController
       def index
         # you can use meta fields from your model instead (e.g. browser_title)
         # by swapping @page for @project in the line below:
-        @dropdown_tags = Refinery::Tags::Tag.where({ name: %w(Residential Commercial Hospitality Institutional Community Mixed-Use)})
-        @adaptive_tag = Refinery::Tags::Tag.find_by_name("Adaptive")
-        @projects = Refinery::Projects::Project.eager_load(:tags,:featured_image)
-
+        @tags = Refinery::Tags::Tag.all.to_a
```
Comment (pksarguru) [inline vendor/extensions/projects/app/controllers/refinery/projects/projects_controller.rb]: @LoganDSPrice why did you add an instance variable?

Comment (LoganDSPrice) [inline vendor/extensions/projects/app/controllers/refinery/projects/projects_controller.rb]: should have just been a local variable

Comment (pksarguru) [inline vendor/extensions/projects/app/controllers/refinery/projects/projects_controller.rb]: cool. do you mind just changing it to a local and then we can merge?

Comment (LoganDSPrice) [inline vendor/extensions/projects/app/controllers/refinery/projects/projects_controller.rb]: already did
